src/features/user_profile/components/SettingsForm/SettingsForm.tsx

```diff
@@ -113,8 +113,9 @@ export function SettingsForm() {
         <div className={s.notFlexParent}>
           <div className={s.flexWrap}>
             <div className={s.profileWrap}>
-              <Text type="heading-xl">{i18n.t('profile.profileSettingsHeader')}</Text>
-
+              <Heading type="heading-06">
```

Comment:
same moment here, padding differs drastically

Reply:
hmm, thanks, i will recheck this

Reply:
fixed